README.md

```diff
@@ -1,121 +1,100 @@
-[Kabit Savaiye Bhai Gurdas Ji Steek](https://drive.google.com/open?id=1-imMygFPdXKj0DuydErsUrtg1IWiu_q7) | [sikhbookclub.com](http://sikhbookclub.com/Book/Kabit-Sawaiye-Bhai-Gurdas-Ji-Steek3)
-[Bhai Nand Lal Granthavali](https://drive.google.com/open?id=1bKjrbqUAn_7TDThLeHcH_GSZuM8iY268) | [sikhbookclub.com](http://sikhbookclub.com/Book/Bhai-Nand-Lal-Granthavali)
-[Rehitname](https://drive.google.com/open?id=1-mJn4UdGMtwb8ELG4VREYU0EPHqXzpu8) | GurbaniNow.com / ShabadOS
-[Amrit Keertan](https://drive.google.com/open?id=1QJdE1ZeutRhkT1FDh1S0b83Z-4rTQyNa) | [sikhbookclub.com](http://sikhbookclub.com/Book/Amrit-Kirtan1) 
-[Sri Dasam Granth Shiromani Steek (Vol. 1)](https://drive.google.com/open?id=10V2Ah4i5s1m_sxpMvFp_uxwVAzHEPwcl)<br> [Sri Dasam Granth Shiromani Steek (Vol. 2)](https://drive.google.com/open?id=1dK3b13mznXq2c88dywk4A5VP2PzBpmhU)<br> [Sri Dasam Granth Shiromani Steek (Vol. 3)](https://drive.google.com/open?id=1w9dvC0M2G1_hRmEDoxZj3aNakK1zv-9W)<br> [Sri Dasam Granth Shiromani Steek (Vol. 4)](https://drive.google.com/open?id=1mxqCLhrCuazeCsstYZUGc2UmW8-TButx)<br> [Sri Dasam Granth Shiromani Steek (Vol. 5)](https://drive.google.com/open?id=1ipiQa7gSQtt5zrRKK-sT0X1kHaStr7kU)<br> [Sri Dasam Granth Shiromani Steek (Vol. 6)](https://drive.google.com/open?id=16R0IrpxnMFBJZ-wvnt0DQ_czrTIqxyXC) | Vol. 1: [discoversikhism.com](http://www.discoversikhism.com/sikh_library/gurmukhi/dasam_sri_guru_granth_sahib_ji_shiromani_steek_1.html)<br> Vol. 2: [discoversikhism.com](http://www.discoversikhism.com/sikh_library/gurmukhi/dasam_sri_guru_granth_sahib_ji_shiromani_steek_2.html)<br> Vol. 3: [discoversikhism.com](http://www.discoversikhism.com/sikh_library/gurmukhi/dasam_sri_guru_granth_sahib_ji_shiromani_steek_3.html)<br> Vol. 4: [discoversikhism.com](http://www.discoversikhism.com/sikh_library/gurmukhi/dasam_sri_guru_granth_sahib_ji_shiromani_steek_4.html)<br> Vol. 5: [discoversikhism.com](http://www.discoversikhism.com/sikh_library/gurmukhi/dasam_sri_guru_granth_sahib_ji_shiromani_steek_5.html)<br> Vol. 6: [discoversikhism.com](http://www.discoversikhism.com/sikh_library/gurmukhi/dasam_sri_guru_granth_sahib_ji_shiromani_steek_6.html) 
-[Sanpuran Steek Sri Sarabloh Granth Ji](https://drive.google.com/open?id=1-C8E3YYeAFIqacfe8Xy7scDFpTyn62_U) | [adelaidesangat.com](http://www.adelaidesangat.com/index.php/downloads/Gurbani-PDF/Sri-Sarabloh-Granth-Sahib/Complete-Sri-Sarbloh-Granth-Sahib-Ji-Steek.pdf/)
-[Sanpuran Steek Sri Sarabloh Granth Ji (Vol. 1)](https://drive.google.com/open?id=1-dbLAUCw2ME4gyBXS7QttId9WxDEkQnf)<br> [Sanpuran Steek Sri Sarabloh Granth Ji (Vol. 2)](https://drive.google.com/open?id=1ooERm2R6rwtedMTs1z21HakqtvHxMqjr) | Vol. 1: [panjabdigilib.org](http://panjabdigilib.org/webuser/searches/displayPage.jsp?ID=9210&page=1&CategoryID=1&Searched=)<br> Vol. 2: [panjabdigilib.org](http://panjabdigilib.org/webuser/searches/displayPage.jsp?ID=9209&page=1&CategoryID=1&Searched=)
```

Comment:
Where does this information go now? Is it in database docs too?

Reply:
it's in [database readme already](https://github.com/ShabadOS/database#gurbani-and-panthic-compositions) and should be updated in the tutorial as well